### PR TITLE
[video_player] audio asset check

### PR DESCRIPTION
**Pull request by INDAPP**
## Description

Support for playing audio files on iOS

## Related Issues

Fixes flutter/flutter#39845
Fixes flutter/flutter#48053

## Checklist

- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by INDAPP** [comment]
@googlebot I fixed it

**Comment by INDAPP** [comment]
@googlebot I fixed it.

**Comment by INDAPP** [comment]
@googlebot I fixed it.

**Comment by INDAPP** [comment]
@googlebot I fixed it.

**Comment by INDAPP** [comment]
@googlebot I fixed it.

File: packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m [hunk 1]
```diff
@@ -308,8 +308,9 @@ - (void)sendInitialized {
     CGFloat width = size.width;
     CGFloat height = size.height;
 
-    // The player has not yet initialized.
-    if (height == CGSizeZero.height && width == CGSizeZero.width) {
+    // The player has not yet initialized for video asset.
+    if (height == CGSizeZero.height && width == CGSizeZero.width &&
```
Comment (Salakar) [inline packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m:312]:
```suggestion
    if ((height == CGSizeZero.height && width == CGSizeZero.width) ||
```

Shouldn't this be an `OR` to preserve the original case?

Comment (Salakar) [reply packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m:312]: Ignore, original is correct.

**Review by Salakar — APPROVED** [review]

**Review by russellwheatley — APPROVED** [review]
LGTM

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks for the contribution! We apologize for the long delay in triaging this PR. We’re in the process of overhauling our PR triage system to respond much more quickly, as well as working through the backlog.

Per Flutter policy, this will need test coverage before we could move forward with the PR. In this case, that would likely mean a new integration test for a file type that currently fails, but passes with this PR.

**Comment by stuartmorgan** [comment]
@ShivamGoyal1899 @russellwheatley Please note that Flutter's policies explicitly say that only committers should use the "approve" feature for PRs; ignoring that just causes confusion for others following a PR.

**Comment by stuartmorgan** [comment]
Thank you for your contribution. I'm going to close this PR for now since there are outstanding comments, just to get this off our PR review queue. Please don't hesitate to submit a new PR if you have the time to address the review comments. Thanks!

**Pull request closed without merging**